cmd/kubeadm/app/cmd/reset.go

```diff
@@ -295,3 +311,19 @@ func resetConfigDir(configPathDir, pkiPathDir string) {
 		}
 	}
 }
+
+//Checks if is control plane
+func isControlPlane(cfg *kubeadmapi.InitConfiguration) bool {
+
```

Comment:
Extra blank lines at start and end of function; should be dropped to match the style of the rest of the file.